packages/react-components/src/components/MessageThread.tsx

```diff
@@ -351,7 +351,7 @@ const getLastChatMessageIdWithStatus = (messages: Message[], status: MessageStat
 const getLastChatMessageForCurrentUser = (messages: Message[]): ChatMessage | undefined => {
   for (let i = messages.length - 1; i >= 0; i--) {
     const message = messages[i];
-    if (message.messageType === 'chat' && message.mine) {
+    if (message?.messageType === 'chat' && message.mine) {
```

Comment:
And here

Reply:
https://github.com/Azure/communication-ui-library/pull/5266#discussion_r1793967977